src/main/kotlin/com/spectralogic/rioclient/RioClient.kt

```diff
@@ -45,8 +47,12 @@ class RioClient(
     username: String = "spectra",
     password: String = "spectra",
     private val requestTimeout: Long = 60L * 1000L, // 60 seconds
+    longLivedToken: String? = null,
+    private val verbose: Boolean = false
```

Comment:
Do we want to set this in the constructor?
If possible a better solution would be to allow caller to decide which calls are verbose and which aren't.
In the case of RioCruise, we could turn on/off verbose logging while a migration is running.

Reply:
How about NOT including verbose in the constructor, and then providing method(s) to toggle it.

Reply:
I think I need to know prior to building the httpClient. By the time they can be toggled I think it is too late. Although maybe I'm not seeing it.

Reply:
Nope, we do it in RioBroker.  
LoggingController.setRootLoggingLevel() --> LoggingUtil.requestLoggingLevelChange()